### ISSUE TITLE
migrate npm-run-all2 to read-package-json-fast

### ISSUE THREAD

**Issue by 43081j** [43081j/ecosystem-cleanup#19]
npm-run-all2 is a fork of npm-run-all:
https://github.com/bcomnes/npm-run-all2

it was created since npm-run-all isn't actively maintained anymore

it depends on `read-pkg` which is fairly bloated. We can switch to npm's own read-package-json-fast:
https://github.com/npm/read-package-json-fast

Both are only for reading the `package.json` in the current working directory, _they do not traverse up the directory tree_.

So we should be able to replace with:

```ts
const data = await readPackageJsonFast('./package.json');

// or

const packagePath = path.join(cwd, './package.json'); // `cwd` is passed in for npm-run-all at least
const data = await readPackageJsonFast(packagePath);
```

It is probably also worth moving people from npm-run-all to npm-run-all2 if we ever bump into that in other repos.

**Comment by 43081j** [comment]
opened bcomnes/npm-run-all2#130 to tackle this

**Comment by 43081j** [comment]
Fixed by bcomnes/npm-run-all2#130 🥳
